### PR TITLE
Convert from Newtonsoft to STJ in cohosting

### PR DESCRIPTION
**Pull request by davidwengier**
Spotted by @DustinCampbell

### CODE REVIEW

File: src/Tools/ExternalAccess/Razor/Cohost/RazorCohostLanguageClient.cs
Comment (davidwengier) [inline src/Tools/ExternalAccess/Razor/Cohost/RazorCohostLanguageClient.cs:65]: This whole file is unused, so this change was quite unnecessary, but figured it would be nicer to paint a clearer picture. https://github.com/dotnet/roslyn/pull/73040 removes the file entirely, just have to wait for VS Preview to ship before we can merge.

**Review by dibarbet — APPROVED** [review]

**Review by DustinCampbell — APPROVED** [review]

**Pull request merged**
